views/default/input/access.php

```diff
@@ -48,6 +48,17 @@
 }
 
 if (is_array($vars['options_values']) && sizeof($vars['options_values']) > 0) {
+
+	if ($vars['value'] && !array_key_exists($vars['value'], $vars['options_values']) && $container && $container->canEdit()) {
```

Comment:
New PR here #7898, you already fixed part of the problem, so no need for the changes to input/access anymore